File: server/router/normal/gost_client_host.go
package normal

import (
	"github.com/gin-gonic/gin"
	"server/controller/normal/gost_client_host"
	"server/global"
	"server/router/middleware"
)

func InitGostClientHost(group *gin.RouterGroup) {
	g := group.Group("gost/client/host", middleware.Auth(global.Jwt))
	g.POST("create", gost_client_host.Create)
	g.POST("admission", gost_client_host.Admission)
	g.POST("renew", gost_client_host.Renew)
	g.POST("update", gost_client_host.Update)
	g.POST("enable", gost_client_host.Enable)
	g.POST("delete", gost_client_host.Delete)
	g.POST("page", gost_client_host.Page)
}

File: web/src/api/normal/gost_client_host.js
import request from "../../request/index.js";

const baseUrl = '/v1/normal/gost/client/host'

export const apiNormalGostClientHostList = () => {
    return request.request({
        url: `${baseUrl}/list`,
        method: 'POST',
    })
}

export const apiNormalGostClientHostPage = (data) => {
    return request.request({
        url: `${baseUrl}/page`,
        method: 'POST',
        data
    })
}

export const apiNormalGostClientHostCreate = (data) => {
    return request.request({
        url: `${baseUrl}/create`,
        method: 'POST',
        data
    })
}

export const apiNormalGostClientHostDomain = (data) => {
    return request.request({
        url: `${baseUrl}/domain`,
        method: 'POST',
        data
    })
}

export const apiNormalGostClientHostRenew = (data) => {
    return request.request({
        url: `${baseUrl}/renew`,
        method: 'POST',
        data
    })
}

export const apiNormalGostClientHostMigrate = (data) => {
    return request.request({
        url: `${baseUrl}/migrate`,
        method: 'POST',
        data
    })
}

export const apiNormalGostClientHostEnable = (data) => {
    return request.request({
        url: `${baseUrl}/enable`,
        method: 'POST',
        data
    })
}

export const apiNormalGostClientHostUpdate = (data) => {
    return request.request({
        url: `${baseUrl}/update`,
        method: 'POST',
        data
    })
}


export const apiNormalGostClientHostDelete = (data) => {
    return request.request({
        url: `${baseUrl}/delete`,
        method: 'POST',
        data
    })
}

export const apiNormalGostClientHostAdmission = (data) => {
    return request.request({
        url: `${baseUrl}/admission`,
        method: 'POST',
        data
    })
}